timeseries/src/autogluon/timeseries/models/local/statsforecast.py

```diff
@@ -19,11 +19,11 @@ def _update_local_model_args(self, local_model_args: Dict[str, Any]) -> Dict[str
         local_model_args["season_length"] = seasonal_period
         return local_model_args
 
-    def _get_model_type(self) -> Type:
+    def _get_model_type(self, version: Optional[str] = None) -> Type:
```

Comment:
`version` signals something like a version number which will be incremented to me. Should we name it something like `method` or `model_type` (the tsintermittent package calls it `type`).

Reply:
Good point! Changed this to `variant` to avoid the confusion with the `model_type` being an instance of `typing.Type`